sql/plan/prepare.go

```diff
@@ -0,0 +1,167 @@
+// Copyright 2020-2022 Dolthub, Inc.
```

Comment:
(super minor) For new source files, just start with the current year for the copyright header comment. 

```suggestion
// Copyright 2022 Dolthub, Inc.
```